public/scripts/services/gameStateService.js

```diff
@@ -0,0 +1,26 @@
+angular.module('scrambleApp').
+  factory('gameStateService', [function() {
+    //Service to hold the state of the app across different controllers
+    var _data = {
+      state: 'start',
+      score: 0,
+      word: ''
+    };
+
+    var extend = function(src, dest) {
```

Comment:
would imagine you could just create a function in here instead of doing a var declaration + assignment